README.md

```diff
@@ -10,4 +10,61 @@ I will not be guarenting anything. I'll try to target both bluesky & mastodon, b
 
 ### This is not (and probably won't be for a while) a 100% accurate recreation of the twitter API
 
-most of this readme todo
\ No newline at end of file
+most of this readme todo
+
+## Compatibility
+**iOS:** 4.2.1 works
+
+**Android:** Untested
+
+**Windows Phone 8/10:** Untested 
+
+**BlackBerry:** Untested
+
+**Nokia Series 40:** Untested
+
+**Windows 8 Metro:** Untested
+
+**Windows 10 UWP:** Untested, lost media?
+
+**macOS:** Untested, lost media
+
+**Tizen Mobile:** Untested, unconfirmed existance
+
+**Apple Watch:** Untested
+
+**Samsung Orsay:** Untested
+
+**Tizen TV:** Untested
+
+**LG NetCast:** Untested
+
+**LG webOS:** Untested, lost
```

Comment:
Does this ever exist / was this named webOS when the client was released?

Reply:
This is unconfirmed client, but was for LG webOS TVs in the early days